index.html

```diff
@@ -3,58 +3,166 @@
-            </p>
+      <a class="navbar-brand" href="/">
+        Cryptohagen
+      </a>
+      <ul class="nav navbar-nav">
+        <li class="nav-item active">
+          <a class="nav-link" href="#">Hvem</a>
+        </li>
+        <li class="nav-item active">
+          <a class="nav-link" href="#when">Hvornår</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#where">Hvor</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#links">Links</a>
+        </li>
+      </ul>
+    </nav>
+    
+    <section id="intro">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-8">
+            <h1>Hvem er det for?</h1>
+            <hr>
             <p>
-              Hvem ved, måske er det dig, der i næste måned sidder og hjælper nye i gang med de ting, du selv lærte på Cryptohagen :-)
+              Cryptohagen er uformelle og hyggelige søndage (hver sidste i måneden), hvor du kan lære at kryptere, og dermed sikre, din kommunikation på internettet.
             </p>
-            <p>Det eneste, du skal have med er godt humør og din bærbare.</p>
-        </div>
-        <div class="col-md-4">
-          <div class="panel panel-default">
-            <div class="panel-heading">
-              Næste Cryptohagen
+              <p>
+                Alle er velkomne, uanset erfaringsniveau eller tekniske kundskaber. Har du brug for hjælp til at komme i gang med at kryptere dine emails? Så er Cryptohagen stedet, du kan lære det. Eller måske ved du alt om GnuPG, eller hvordan man får OTR til at virke over IRC? Så skal du også være velkommen til at dukke op og hjælpe andre i gang.
+              </p>
+              <p>
+                Hvem ved, måske er det dig, der i næste måned sidder og hjælper nye i gang med de ting, du selv lærte på Cryptohagen :-)
+              </p>
+              <p>Det eneste, du skal have med er godt humør og din bærbare.</p>
+            </div>
+          </div>
+      </div>
+    </section>
+    <section id="intro">
+     
+      <div class="container">
+        <h1>Næste gang!</h1>
+        <hr>
+      
+        <div class="card-columns">
+
+          <div class="card card-inverse card-primary">
+            <div class="card-block">
+              <h5 class="card-title">Søndag d. 31. juli 2016</h5>
+              <p class="card-text">
+                Cryptohagen løber af stablen igen! 
+                Vi sidder i <strong>Mellemrummet</strong>! 
+                Kom og drik god fairtradekaffe med os og lær at sikre dit privatliv på det vilde web!
+              </p>
+              <p class="card-text">
+                <strong>Tidspunkt</strong><br />
+                13:00-17:00
+              </p>
```

Comment:
..or what I meant was that I tried to see if I could do it in 5 minutes and failed :)